languages/python/exercises/concept/elyses-enchantments/.docs/hints.md

```diff
@@ -0,0 +1,70 @@
+## General
+
+## 1. Creating a List
+
+- `Lists` in python may be [constructed](https://docs.python.org/3/library/stdtypes.html#list) in several ways.
+- If you want to create a `list` with a small amount of elements, the easiest way to do it is to use the square brackets and a comma to separate two elements. For example, to create a `list` containing the elements a, b and c, you can write:
+
+```python
+>>> ['a', 'b', 'c']
+```
+
+- Another way to create a `list` is to use the type constructor `list()`. To create the same list as in the previous example, you can write:
+
+```python
+>>> list(('a', 'b', 'c'))
+
+```
+
+## 2. Creating a copy of a List
+
+- `Lists` can be [nested](https://realpython.com/python-lists-tuples/#lists-can-be-nested), this means that a `list` can be an element of another `list`.
+
+## 3. Concatenating Lists
+
+- Sequence types such as `list` already support a few [common operations](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range)
+
+## 4. Testing List Membership
+
+- Sequence types such as `list` already support a few [common operations](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range)
+
+## 5. Accessing List Elements by Index
+
+- Sequence types such as `list` already support a few [common operations](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range)
+- Remember that the first element of the `list` is at index 0
+- In python, negative indexing starts the count from the end. This mean that you can find the last element of the `list` at index -1.
+
+## 6. Accessing Sublists by Slicing
+
+- Sequence types such as `list` already support a few [common operations](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range)
+- For the last part of the exercise, think about reusing the code from the functions that you just implemented.
+
+## 7. Iterating Over List Items
+
+- There are a few techniques to [iterate over a `list` in python](https://www.geeksforgeeks.org/iterate-over-a-list-in-python/).
+- The easiest one is the for loop, here is an example of how to use it:
+
+```python
+>>> list = [1, 2, 3, 4, 5]
+>>> for item in list:
+...     item*2
+>>>list
+[2, 4, 6, 8, 10]
+```
+
```

Comment:
I really like this example, and the `hints.md` file overall.  But I am a bit "on the fence".  These are _hints_, and so I don't know if we want specific code examples.  @valentin-p @cmccandless  your thoughts here & for the rest of the file?

Reply:
Thank you, I tried my best but I wasn't sure how precise I should be because I know that they would only use the hint in last resort but at the same time this subject is not too difficult and it is well explained in the python docs.

Reply:
I agree that explaining in plain English would be more suitable.

See for example: [dicts hints.md](https://github.com/exercism/v3/blob/master/languages/python/exercises/concept/inventory-management/.docs/hints.md)

Reply:
I updated the file, could you check if it is better now? Thanks

Reply:
Thank you for all the work, including fixing little things that you did not write yourself. That's perfect.